python/paddle/fluid/layers/control_flow.py

```diff
@@ -1724,6 +1724,9 @@ def shrink_memory(x, i, table):
         usage.
     """
     helper = LayerHelper('shrink_memory', **locals())
+    check_type(x, 'x', (Variable), 'shrink_memory')
```

Comment:
Why do you put "()" around Variable? Suggest to use `Variable` or `(Variable, )` instead

Reply:
Done.